packages/lambda-functions/env-example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "HelloWorldFunction": {
+    "DATABASE_URL": ""
+  }
+}
```

Comment:
If you copy this file to `env.json` and set your connection string using `host.docker.internal` instead of localhost, this file will be read by SAM to set the environment of the lambda in the docker container